metricbeat/docs/modules/kibana.asciidoc

```diff
@@ -3,10 +3,9 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-module-kibana]]
-== kibana Module
-
-This is the kibana Module.
+== Kibana Module
 
+This is the Kibana Module and only tracks the high level metrics. To monitor more Kibana metrics, use our {monitoringdoc}/xpack-monitoring.html[X-Pack monitoring] which is available under a free basic license.
```

Comment:
@ruflin what high level metrics are we tracking? can you point me to a PR/issue?